.vscode/settings.json

```diff
@@ -0,0 +1,4 @@
+{
+    "python-envs.defaultEnvManager": "ms-python.python:system",
+    "python-envs.pythonProjects": []
+}
```

Comment:
What is this code for? Is it intentional? Or should we add `.vscode` to `.gitignore`?